.vscode/tasks.json

```diff
@@ -6,7 +6,7 @@
 		{
 			"label": "export",
 			"type": "shell",
```

Comment:
btw you can simplified this to `mkdir -p file-icons product-icons`

Reply:
resolved in https://github.com/mightbesimon/vscode-emoji-icons/commit/89b804e097c4e20f6f5d521b8a1b109c5977694a